File: SearchAlgos/Linear Search/linearSearch.c
#include <stdio.h>

int searchAlgo(int arr[], int len, int num){
    for(int i = 0; i < len; i++){
        if(arr[i] == num){
            return i;
        }
    }
    // This check is to avoid any errors that 
    // may occur if the number is not present in the array
    return -1;
}

void main(){
    int arr[] = {12, 4, 54, 81, 10, 1};
    int num = 12;
    int len = sizeof(arr) / sizeof(arr[0]); 
    int result = searchAlgo(arr, len, num);

    if(result != -1){
        printf("%d is found at %d", num, result);
    }
    else{
        printf("Number is not found in the array");
    }
}

File: SearchAlgos/Linear Search/linearSearch.py
def searchAlgo(arr, num): 
    for i in arr:
        if (arr[i]== num):
            return i
    # This is a check to avoid error 
    # incase number is not present in the array
    return -1

array = [1, 8, 7, 9, 5, 6]
num1 = 8

results = searchAlgo(array, num1)

if results != -1:
    print(num1, "found at index: ", results);
else:
    print("Number not present in array");